src/H5VL_log_dataset.cpp

```diff
@@ -387,18 +377,32 @@ herr_t H5VL_log_dataset_read_2 (size_t count,
                                 void *buf[],
                                 void **req) {
     herr_t err = 0;
+    H5VL_log_dset_t *dp       = (H5VL_log_dset_t *)dset;
+    H5VL_log_dset_info_t *dip = NULL;  // Dataset info
+    hid_t dsid;                        // Dataset space id
+    H5VL_log_selections *dsel = NULL;  // Selection blocks
```

Comment:
It looks like all these new variables above are unused here.

Reply:
Fixed.